api/v1beta1/moduleimagesconfig_types.go

```diff
@@ -72,6 +72,11 @@ type ModuleImagesConfigSpec struct {
 	// ImagePullPolicy defines the pull policy used for verifying the presence of the image
 	//+optional
 	ImagePullPolicy v1.PullPolicy `json:"imagePullPolicy"`
+
+	// Boolean flag that determines whether images built must also
+	// be pushed to a defined repository
+	// +optional
+	PushBuiltImage bool `json:"pushBuiltImage"`
```

Comment:
Let's put it in `images[]`? Alongside with `registryTLS` and `skipWaitMissingImage`.

Reply:
But we are building each image individually, and each one of them can be pushed or not. The same way that we allow different TLS conf for each image separately and decide for each image if we skip waiting or not.

The reason `imageRepoSecret` is global because it is like a single `dockerconfigjson` file on a Linux FS that contains tokens for all registries.

By making `pushBuiltImage` global, we restrict the user while other similar configuration are allowed per image.

Reply:
Since the `Preflight` CRD doesn't have this kind of granularity, we will make this `pushBuildImage` field global.